db/migrate/20240210054714_devise_create_usuarios.rb

```diff
@@ -1,8 +1,8 @@
 # frozen_string_literal: true
 
-class DeviseCreateUsuarios < ActiveRecord::Migration[7.1]
+class DeviseCreateUsers < ActiveRecord::Migration[7.1]
```

Comment:
Previous migrations shouldn't be modified.
The reason is that, when you want to clone your repository in another computer and run the migrations, a `users` table will be created in this migration and when you run the next migration to rename the table from `usuarios` to `users` it won't find the `usuarios` table.
So, please leave this migration as it was.

Reply:
D:! oh no!